test/studies/shootout/k-nucleotide/benmcd/knucleotide-user-hash.chpl

```diff
@@ -0,0 +1,149 @@
+use IO, Map, Sort;
+
+config param tableSize = 2**16,
+             columns = 61;
+
+record intWrapper {
+  var val: int;
+  proc init() {
+    val = 0;
+  }
+  proc init(i: int) {
+    val = i;
+  }
```

Comment:
It doesn't seem to me like either of these initializers should be required (they mimic what the default initializer does), so I'd just remove them to reduce your code size and complexity (esp. since we graph code size on our scatter plots).